source/features/gha-runs-from-workflow-file.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const isWorkflowFile = (): boolean => ( ((location.pathname.endsWith('.yaml')) || (location.pathname.endsWith('.yml'))) && location.pathname.split('/', 7)[5] == ".github"  && location.pathname.split('/', 7)[6] == "workflows");
+
+function init(): void {
+  
+  const actionName = 
+	select.all('.blob-code-inner')
+		.find(line => line.textContent.startsWith('name'))
+		.textContent
+		.replace(/^name:\s+/, '')
+		.replace('"', '')
+		.replace("'", '')
```

Comment:
`.replace` with a string will only replace the first occurrence; you'll need a regex with the `g` flag
```suggestion
		.replace(/["']/g, '')
```